bundles/org.openhab.binding.tapocontrol/README.md

```diff
@@ -0,0 +1,97 @@
+# TapoControl Binding
+
+This binding adds support to control Tapo (Copyright © TP-Link Corporation Limited) Smart Home Devices from your local openHAB system.
+(Note: This is not an official Tapo/TP-link Binding!)
```

Comment:
If this is merged, it *will be* an official binding. If you mean this is not made by TP-Link, you don't need to mention it as this is the normal case.